backend/benefit/azure-pipelines-master.yml

```diff
@@ -0,0 +1,47 @@
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push 
+# an update to the specified branches or you push specified tags.
+trigger:
+  branches:
+    include:
+    - develop
+  paths:
+    exclude:

```

Comment:
https://github.com/City-of-Helsinki/yjdh/blob/develop/backend/kesaseteli/azure-pipelines-kesaseteli-api-master.yml#L10